src/Writer/Writer.php

```diff
@@ -25,8 +25,8 @@ public function __construct(string $file = null)
 
     public function save(): bool
     {
-        if (!is_null($this->file) && file_exists($this->file) && is_writable($this->file)) {
-            $result = file_put_contents($this->file, $this->text);
+        if (!is_null($this->file)) {
+            $result = @file_put_contents($this->file, $this->text);
```

Comment:
I agree with you but not related to OUTPUTFORMAT.
I'll refactor the `parse()`, `write()`, and `save()` function in a next PR.
Please revert.